frontend/__tests__/components/actions/items-action.test.ts

```
@@ -0,0 +1,182 @@
+
+// 辅助函数：重置模块缓存
+function clearItemsActionCache() {
+  // 通过重新导入模块并手动重置缓存
+  const itemsActionModule = require("@/components/actions/items-action");
+  // 访问模块内部的缓存变量并重置
+  if (itemsActionModule && typeof itemsActionModule === 'object') {
+    // 直接访问模块的内部状态（虽然不是最佳实践，但这是测试需要）
+    eval(`
+      const itemsActionPath = require.resolve("@/components/actions/items-action");
+      if (require.cache[itemsActionPath]) {
+        // 通过直接修改模块代码中的缓存变量
```

Biome (1.9.4)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)